Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
+++ /dev/null	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
Index: BackEnd/gradle/wrapper/gradle-wrapper.properties
===================================================================
diff --git a/BackEnd/gradle/wrapper/gradle-wrapper.properties b/BackEnd/gradle/wrapper/gradle-wrapper.properties
deleted file mode 100644
--- a/BackEnd/gradle/wrapper/gradle-wrapper.properties	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
+++ /dev/null	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
@@ -1,7 +0,0 @@
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
-networkTimeout=10000
-validateDistributionUrl=true
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
Index: BackEnd/settings.gradle
===================================================================
diff --git a/BackEnd/settings.gradle b/BackEnd/settings.gradle
deleted file mode 100644
--- a/BackEnd/settings.gradle	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
+++ /dev/null	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
@@ -1,1 +0,0 @@
-rootProject.name = 'BackEnd'
Index: BackEnd/README.md
===================================================================
diff --git a/BackEnd/README.md b/BackEnd/README.md
deleted file mode 100644
--- a/BackEnd/README.md	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
+++ /dev/null	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
@@ -1,1 +0,0 @@
-
Index: BackEnd/build.gradle
===================================================================
diff --git a/BackEnd/build.gradle b/BackEnd/build.gradle
deleted file mode 100644
--- a/BackEnd/build.gradle	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
+++ /dev/null	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
@@ -1,37 +0,0 @@
-plugins {
-    id 'java'
-    id 'org.springframework.boot' version '3.4.4'
-    id 'io.spring.dependency-management' version '1.1.7'
-}
-
-group = 'com.example'
-version = '0.0.1-SNAPSHOT'
-
-java {
-    toolchain {
-        languageVersion = JavaLanguageVersion.of(17)
-    }
-}
-
-configurations {
-    compileOnly {
-        extendsFrom annotationProcessor
-    }
-}
-
-repositories {
-    mavenCentral()
-}
-
-dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    compileOnly 'org.projectlombok:lombok'
-    annotationProcessor 'org.projectlombok:lombok'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
-    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
-}
-
-tasks.named('test') {
-    useJUnitPlatform()
-}
Index: BackEnd/src/main/java/com/example/backend/BackEndApplication.java
===================================================================
diff --git a/BackEnd/src/main/java/com/example/backend/BackEndApplication.java b/BackEnd/src/main/java/com/example/backend/BackEndApplication.java
deleted file mode 100644
--- a/BackEnd/src/main/java/com/example/backend/BackEndApplication.java	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
+++ /dev/null	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
@@ -1,13 +0,0 @@
-package com.example.backend;
-
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-
-@SpringBootApplication
-public class BackEndApplication {
-
-    public static void main(String[] args) {
-        SpringApplication.run(BackEndApplication.class, args);
-    }
-
-}
Index: BackEnd/src/test/java/com/example/backend/BackEndApplicationTests.java
===================================================================
diff --git a/BackEnd/src/test/java/com/example/backend/BackEndApplicationTests.java b/BackEnd/src/test/java/com/example/backend/BackEndApplicationTests.java
deleted file mode 100644
--- a/BackEnd/src/test/java/com/example/backend/BackEndApplicationTests.java	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
+++ /dev/null	(revision 45c7791c2c04e20d6b3bb346b4ab7d4acd029305)
@@ -1,13 +0,0 @@
-package com.example.backend;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class BackEndApplicationTests {
-
-    @Test
-    void contextLoads() {
-    }
-
-}
Index: BackEnd/src/main/resources/static/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BackEnd/src/main/resources/static/index.html b/BackEnd/src/main/resources/static/index.html
new file mode 100644
--- /dev/null	(date 1742996898132)
+++ b/BackEnd/src/main/resources/static/index.html	(date 1742996898132)
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+  <h1>Hello World</h1>
+</body>
+</html>
\ No newline at end of file
Index: BackEnd/src/main/java/com/stravacopy/backend/HomeController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BackEnd/src/main/java/com/stravacopy/backend/HomeController.java b/BackEnd/src/main/java/com/stravacopy/backend/HomeController.java
new file mode 100644
--- /dev/null	(date 1742996898126)
+++ b/BackEnd/src/main/java/com/stravacopy/backend/HomeController.java	(date 1742996898126)
@@ -0,0 +1,13 @@
+package com.stravacopy.backend;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class HomeController {
+
+    @RequestMapping("/")
+    public String index(){
+        return "index.html";
+    }
+}
