Index: BackEnd/src/main/resources/static/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BackEnd/src/main/resources/static/index.html b/BackEnd/src/main/resources/static/index.html
new file mode 100644
--- /dev/null	(date 1743160640814)
+++ b/BackEnd/src/main/resources/static/index.html	(date 1743160640814)
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+  <h1>Hello World</h1>
+</body>
+</html>
\ No newline at end of file
Index: BackEnd/src/main/java/com/stravacopy/backend/HomeController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BackEnd/src/main/java/com/stravacopy/backend/HomeController.java b/BackEnd/src/main/java/com/stravacopy/backend/HomeController.java
new file mode 100644
--- /dev/null	(date 1743160640829)
+++ b/BackEnd/src/main/java/com/stravacopy/backend/HomeController.java	(date 1743160640829)
@@ -0,0 +1,13 @@
+package com.stravacopy.backend;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class HomeController {
+
+    @RequestMapping("/")
+    public String index(){
+        return "index.html";
+    }
+}
